src/Protocol/Diaspora.php

```diff
@@ -4051,6 +4051,29 @@ public static function sendProfile($uid, $recips = false)
 			return;
 		}
 
+		$r = dba::p("SELECT
```

Comment:
Wouldn't it be better to move this into the ```Contact``` class as method? We are using this very often.

Reply:
Your review is already outdated, and I put it in `User` because we're using a user id.